pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--Please fill in the **bold** fields, and tick all applicable boxes.-->
+
+Added **your URL here**.
```

Comment:
No, I meant your `Added **your URL here**.` text should be a comment, but doesn't matter. We can leave it like this.